internal/manifests/collector/rbac.go

```diff
@@ -50,7 +64,16 @@ func ClusterRole(params manifests.Params) *rbacv1.ClusterRole {
 }
 
 func ClusterRoleBinding(params manifests.Params) *rbacv1.ClusterRoleBinding {
-	configFromString, err := adapters.ConfigFromString(params.OtelCol.Spec.Config)
+	otelCol := convert.V1Alpha1to2(params.OtelCol)
```

Comment:
same as above... I do think this may be easier for us to just do the conversion when we construct manifest.Params

Reply:
https://github.com/open-telemetry/opentelemetry-operator/pull/2532#discussion_r1458052336